assets/state-dra-driver/0700_daemonset.yaml

```diff
@@ -0,0 +1,130 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    app: nvidia-dra-driver-kubelet-plugin
+  name: nvidia-dra-driver-kubelet-plugin
+  namespace: "FILLED BY THE OPERATOR"
+spec:
+  selector:
+    matchLabels:
+      app: nvidia-dra-driver-kubelet-plugin
+  updateStrategy:
+    rollingUpdate:
+      maxSurge: 0
+      maxUnavailable: 1
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: nvidia-dra-driver-kubelet-plugin
+    spec:
+      priorityClassName: system-node-critical
+      serviceAccountName: nvidia-dra-driver
+      # TODO: revisit the affinity / nodeSelector for this daemonset
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: feature.node.kubernetes.io/pci-10de.present
+                operator: In
+                values:
+                - "true"
+            - matchExpressions:
+              - key: feature.node.kubernetes.io/cpu-model.vendor_id
+                operator: In
+                values:
+                - NVIDIA
+            - matchExpressions:
+              - key: nvidia.com/gpu.present
+                operator: In
+                values:
+                - "true"
+      initContainers:
+      - image: "FILLED BY THE OPERATOR"
+        name: driver-validation
+        command: [ 'sh', '-c' ]
+        args: [ "until [ -f /run/nvidia/validations/driver-ready ]; do echo waiting for driver to be setup; sleep 5; done" ]
+        securityContext:
+          privileged: true
+        volumeMounts:
+        - name: run-nvidia-validations
+          mountPath: /run/nvidia/validations
+          mountPropagation: HostToContainer
+      containers:
+      - name: plugin
+        image: "FILLED BY THE OPERATOR"
+        imagePullPolicy: IfNotPresent
+        command: ["/bin/bash", "-c"]
+        args:
+          - /bin/entrypoint.sh
+        env:
+        - name: MASK_NVIDIA_DRIVER_PARAMS
+          value: "false"
```

Comment:
Question: Can a user still override this if required?

Reply:
Yes. The user can override this variable in Clusterpolicy with `draDriver.env`.